docs/index-utils.md

```diff
@@ -44,6 +51,18 @@ Dumps raw document.
 _NOTICE:_ available only if the raw documents are stored with indexing option `-storeRawDocs`
 
 
+```
+-dumpRawDocs docidsInputPath
```

Comment:
what about `-dumpRawDocsWithDocid` and `-dumpRawDocsWithoutDocid`?

Reply:
sg

Reply:
Or `-dumpRawDocsWithDocid` and `-dumpRawDocs`?
Anserini by default does not include docid in the raw doc